src/components/NavButtons.js

```diff
@@ -28,9 +31,9 @@ function NavButtons(props) {
 
     return (
         <div className="buttons">
-            { props.currentPage > PAGE.START && ( props.currentPage < PAGE.RESULT || !props.issueUrl.validity ) && <button type="button" className="button button--green" name="prev" onClick={onNavBtnClick}>Prev</button> }
-            { props.currentPage < PAGE.SUBMIT && <button type="button" className="button button--green" name="next" disabled={!completed} onClick={onNavBtnClick}>Next</button> }
-            { props.currentPage == PAGE.SUBMIT && <button type="button" className="button button--green" name="submit" disabled={process.env.NODE_ENV === 'production' ? !props.captchaResponse.validity : false} onClick={onSubmitBtnClick}>Submit</button> }
+            { props.currentPage > PAGE.START && ( props.currentPage < PAGE.RESULT || !props.issueUrl.validity ) && <button type="button" className="button button--green" name="prev" onClick={onNavBtnClick}>{Translator.trans('NavPrev')}</button> }
+            { props.currentPage < PAGE.SUBMIT && <button type="button" className="button button--green" name="next" disabled={!completed} onClick={onNavBtnClick}>{Translator.trans('NavNext')}</button> }
+            { props.currentPage == PAGE.SUBMIT && <button type="button" className="button button--green" name="submit" disabled={process.env.NODE_ENV === 'production' ? !props.captchaResponse.validity : false} onClick={onSubmitBtnClick}>{Translator.trans('NavSubmit')}</button> }
```

Comment:
it will be nice, if you can use scoped translations. scopes is namespaces with dots-spearators.
i think, phrases should contains current step as namespaces or "global" if its global phrase.
e.g.
```json
"global.nav.prev": "",
"global.nav.next": "",
...
"step_1.title":"",
"step_1.subtitle":"",
....
"step_2.title":"",
"step_2.subtitle":"",
"step_2.smth":"",
```